test/unittests/parser_test.cpp

```diff
@@ -17,6 +17,15 @@ bytes prefix_with_size(const bytes& content)
     return bytes{static_cast<uint8_t>(content.size())} + content;
 }
 
+bytes make_vec(std::vector<bytes_view> contents)
```

Comment:
```suggestion
bytes make_vec(std::initializer_list<bytes_view> contents)
```

Reply:
As most of the sections are vectors, you can add overload `make_section(uint8_t id, std::initializer_list<bytes_view> contents)`.

Reply:
Add or replace?

Reply:
If `make_vec` is not used in any other context, I would remove it.

Reply:
I'm strongly in favor of `make_vec` because this is how the wasm spec reference it (i.e. we mean wasm `vec` not C++ `std::vector`).

In the other matter, I'm against adding anything that might be used some day. If it cannot be used now, I prefer not to adding it.

Reply:
Changed from `vector` to `initializer_list`. Kept the name as it reflects the spec.